src/ldc/attributes.d

```diff
@@ -223,3 +223,23 @@ immutable weak = _weak();
 private struct _weak
 {
 }
+
+///Readability aliases for compute
```

Comment:
Done. 

re dmd-test suite: It's always worked for me. In the spec it says 
```
ModuleDeclaration:
    ModuleAttributes_(opt) module ModuleFullyQualifiedName ;

ModuleAttributes:
    ModuleAttribute
    ModuleAttribute ModuleAttributes

ModuleAttribute:
    DeprecatedAttribute
    UserDefinedAttribute
```
and a module decl is a dsymbol and thus has a `UserAttributeDeclaration*` field.
So It is very unlikely to be accidentally broken. 
I'll have a look and see if I can find if it is covered.